tests/smoke-tests/test-ethereum-db-mapping.ts

```diff
@@ -0,0 +1,40 @@
+import { expect } from "chai";
+import { describeSmokeSuite } from "../util/setup-smoke-tests";
+
+const debug = require("debug")("smoke:ethereum-mapping");
+
+const wssUrl = process.env.WSS_URL || null;
+const ethRpcUrl = process.env.HTTPS_URL || null;
+const relayWssUrl = process.env.RELAY_WSS_URL || null;
+
+// At rpc-level, `*ByNumber` requests always use the canonical block reference given by Substrate.
+// In the other hand `*ByHash` requests rely on data mapped in the frontier db.
+// We want to compare both to verify recent db data consistency and rpc impl across client versions.
+
+describeSmokeSuite(
+  `Ethereum secondary DB should contains valid data`,
+  { wssUrl, relayWssUrl, ethRpcUrl },
+  (context) => {
+    it("should get the same response payload on byNumber and byHash requests", async function () {
+      this.timeout(6_000_000); // 30 minutes
+      // As we are testing rpc-level functionality the height at which we access secondary db data
+      // is irrelevant. We can just select some arbitrary block numbers to verify block hashes.
+      const latestBlockNumber = await context.ethers.getBlockNumber();
+      // We asume we only want to run the test in a live chain.
+      if (latestBlockNumber > 10000) {
```

Comment:
Not the best way to check if real live chain, but it is fine for now.
We will probably go through some refactoring of "conditional" test soon